src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OverrideEqualsAndOperatorEqualsOnValueTypes.cs

```diff
@@ -49,13 +50,42 @@ public sealed class OverrideEqualsAndOperatorEqualsOnValueTypesAnalyzer : Diagno
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(EqualsRule, OpEqualityRule);
 
+        protected virtual bool IsAssignableTo(INamedTypeSymbol type, INamedTypeSymbol assignableToType)
+        {
+            // TODO: Use the language specific helper for IsAssignableTo.
```

Comment:
This needs to invoke the new `IsAssignableTo` helper that @lgolding plans to add soon.